src/Core/Action/EntrypointAction.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+/*
+ * This file is part of the API Platform project.
+ *
+ * (c) Kévin Dunglas <dunglas@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace ApiPlatform\Core\Action;
+
+class_exists(\ApiPlatform\Action\EntrypointAction::class);
+
+if (false) {
+    class EntrypointAction extends \ApiPlatform\Action\EntrypointAction
```

Comment:
as mentioned in the internal discussion, since `ApiPlatform\Action\EntrypointAction` is final, the deprecated alias ( `ApiPlatform\Core\Action\EntrypointAction` ) needs to also be declared final so static analysis tools would be able to warn users when they try to extend from it. ( same applies for all other final classes ) 

```suggestion
    final class EntrypointAction extends \ApiPlatform\Action\EntrypointAction
```